src/engine/errors.ts

```diff
@@ -30,6 +31,17 @@ export class LimitsExceededError extends ContentfulError {
   }
 }
 
+export class LimitsExceededForApplyError extends ContentfulError {
```

Comment:
Should we give [the other LimitsExceeded error](https://github.com/contentful/contentful-merge/pull/363/files#diff-47876649927a4e73623e91b40e9227cac585dba2c76453af7e56cd181d3d9d3eR17) also a more explicit name now that we have two?